src/popup/router/components/OverflowableText.vue

```diff
@@ -0,0 +1,30 @@
+<template>
+  <span class="overflowable-text">
+    <span class="text">{{ isName ? isName : text }}</span>
+    <span v-if="isName">{{ `.chain` }}</span>
```

Comment:
```suggestion
    <span class="text">{{ text.substr(0, text.length / 2) }}</span>
    <span>{{ text.substr(text.length / 2) }}</span>
```
Will it work the same as TruncateMid then?